### PR TITLE
cleaned up test database locations

### PR DESCRIPTION
**Pull request by tanaes**
Just cleaning up the test folder and updating configs to match. 

@mortonjt @sjanssen2 @qiyunzhu Mind taking a look and merging?

### CODE REVIEW

File: oecophylla/cli/tests/data/exp_config.yaml [hunk 1]
```diff
@@ -12,18 +12,21 @@ params:
...
   binning_samples:
   - sample_S22282
   - sample_S22205
-  filter_db: test_data/phix
+  mapping_assembler:
+  - metaspades
+  filter_db: test_data/test_dbs/bowtie2/phix
+  trimmer: atropos
+  anvio:
+    centrifuge_base: /home/jgsanders/miniconda/envs/anvio2/centrifuge
```
Comment (sjanssen2) [inline oecophylla/cli/tests/data/exp_config.yaml:26]: are we OK pointing to your home dir?

**Review by sjanssen2 — COMMENTED** [review]
If Travis passes we are good to go

File: oecophylla/cli/tests/data/exp_config.yaml (hunk 1 above)
Comment (tanaes) [reply oecophylla/cli/tests/data/exp_config.yaml:26]: These paths are still legacy and we'll update with the Anvi'o module...

File: oecophylla/cli/tests/data/tool_params.yml [hunk 2]
```diff
@@ -25,39 +24,29 @@ humann2:
...
 metaphlan2:
-  # metaphlan2_dir: /home/jgsanders/git_sw/metaphlan2
-  db: test_data/mp2_test_db/mpa_v20_m200
-
-shogun:
-  db: test_data/shogun_test_db
-  # alignment algorithm: bowtie2, burst and utree
-  aligner: utree
-  # taxonomic ranks (kingdom to strain) to redistribute reads
+  db: test_data/test_dbs/mp2_test_db/mpa_v20_m200
```
Comment (qiyunzhu) [inline oecophylla/cli/tests/data/tool_params.yml:29]: This `mpa_v20_m200` is the original MetaPhlAn2 database name. We will need to replace it with some minimal thing at some time.

**Review by qiyunzhu — APPROVED** [review]

**Comment by qiyunzhu** [comment]
Looks good to me!

**Pull request merged**